docs/components/clerk-provider.mdx

```diff
@@ -94,7 +94,7 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
 | Name                      | Type                                                                                  | Description                                                                                                                                                                                                                |
 | ------------------------- | ------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `publishableKey`          | `string`                                                                              | Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.                                                                                                                                                   |
-| `frontendApi`             | `string`                                                                              | The frontend API host for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page under the **Advanced** section.                                                                                                                                        |
+| `frontendApi` (deprecated)             | `string`                                                                              | Deprecated in favor of `publishableKey`.                                                                                                                                        |

```

Comment:
is this a correct update? or was `frontendApi` being used for the frontend API URL ?

Reply:
`frontendApi` should be removed from this table as it can no longer be used as a prop